middleware/app/Utilities/ProviderUtil.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+use App\Models\ProviderPlatform;
+
+class ProviderUtil
```

Comment:
I don't think we need these wrapper util classes, but we can pull out some of the methods into services.

Reply:
Also possibly rename to ProviderPlatformServices